app/views/static_pages/_last_room_update_log.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="col-lg-4 col-md-6 col-sm-12 mb-3">
+  <div class="card">
+    <div class="card-header">
+      <h3>Resources Script</h3>
+    </div>
+    <div class="card-body">
+      <h6 class="card-text mb-4">Last Run at: <%= show_date_with_time(last_room_update_log.created_at) %></h6>
+      <h6 class="card-text mb-4">
+        Status:
+        <span
+          class="<%= room_update_log_status(last_room_update_log.note) == 'error' ? 'text-danger' : 'text-success' %>">
+          <%= last_room_update_log.note.include?('success') ? 'SUCCESS' : 'ERROR' %>
+        </span>
+      </h6>
+
+      <h6 class="card-text mb-4">
+        Note:
+        <% if room_update_log_status(last_room_update_log.note) == 'success' %>
+          Resources updated successfully for rooms.
+        <% elsif room_update_log_status(last_room_update_log.note) == 'success-partial' %>
+          Resources updated successfully for rooms.
+          <a class="text-primary" data-bs-toggle="collapse" href="#nonExistingRooms" role="button" aria-expanded="false" aria-controls="nonExistingRooms">
+            These
+          </a>
+          rooms (rmrecnbr) don't exist in WebCheckout.
+          <div class="collapse" id="nonExistingRooms">
+            <div class="card card-body">
+              <%= last_room_update_log.note.split(':').last.strip %>
+            </div>
+          </div>
+        <% elsif room_update_log_status(last_room_update_log.note) == 'error' %>
+          An error occurred while updating resources for rooms.
+          <a class="text-primary" href="#myCustomTrigger">Report an issue.</a>
```

Comment:
This link is not working for me, nothing happens when I click it. We don't need a link here, just the words 'Report an issue'